internals/cli/masker/writer.go

```diff
@@ -195,7 +196,7 @@ func (mw *MaskedWriter) Run() {
 		case <-time.After(mw.timeout):
 			// force the buffer to flush if not already done so.
```

Comment:
This comment still refers to a buffer flush. Is that still correct after your renaming to `outputTimeout`?